docs/configuration.rst

```diff
@@ -0,0 +1,49 @@
+===================
+Configuring Drydock
+===================
+
+Drydock uses an INI-like standard oslo_config file. A sample
+file can be generated via tox::
+
+    $ tox -e genconfig
+
+Customize your configuration based on the information below
+
+Keystone Integration
+====================
+
+Drydock requires a service account to use for validating client
+tokens::
+
+    $ openstack domain create 'ucp'
+    $ openstack project create --domain 'ucp' 'service'
+    $ openstack user create --domain ucp --project service --project-domain 'ucp' --password drydock drydock
+    $ openstack role add --project-domain ucp --user-domain ucp --user drydock --project service admin
+
+The service account must then be included in the drydock.conf::
+
+    [keystone_authtoken]
+    auth_uri = http://<keystone_ip>:5000/v3
```

Comment:
I've refactored the config options using the keystoneauth1 module, I'll update this doc accordingly.